src/test/controllers/company.details.controller.spec.unit.ts

```diff
@@ -266,6 +267,61 @@ describe("company.details.controller tests", () => {
     expect(res.text).toContain(GENERIC_ERROR);
     expect(res.text).toContain(TITLE);
   });
+
+  it("should return filing date page when filing date is after configured period after due date", async () => {
+    const dummyCompanyProfile: ExtensionsCompanyProfile = mockUtils.getDummyCompanyProfile(false, true);
+    dummyCompanyProfile.accountingPeriodEndOn = new Date(2021,1,1,0,0,0).toDateString();
+    dummyCompanyProfile.accountsDue = new Date(2022,1,2,0,0,0).toUTCString();
+    mockCompanyProfile.mockResolvedValue(dummyCompanyProfile);
+
+    const res = await request(app).post(EXTENSIONS_CONFIRM_COMPANY)
+      .set("referer", "/")
+      .set("Cookie", [`${COOKIE_NAME}=123`]);
+
+    expect(res.header.location).toEqual(pageURLs.EXTENSIONS_EXTENSION_LIMIT_REACHED);
+    expect(res.status).toEqual(302);
+  });
+
+  it("should return filing date page when filing date is on the configured period after due date", async () => {
+    const dummyCompanyProfile: ExtensionsCompanyProfile = mockUtils.getDummyCompanyProfile(false, true);
+    dummyCompanyProfile.accountingPeriodEndOn = new Date(2021,1,1,0,0,0).toDateString();
```

Comment:
add some times to the dates so we know that time stripping is working. Make it so that it would fail if the time stripping didn't happen (due date > periodEndOn?  or the other way around)

Reply:
Added times so that if time stripping failed the tests would fail.